assignment7.txt

```diff
@@ -0,0 +1,25 @@
+Ava N.
+July 7, 2015
+
+Assignment 7
+
+1. ++*p increments value at location p point to, then evaluates (stores) to incremented value.
+*p++ evaluates the value at location p points to, then advances p. (*p)++ evaluates the value at the location p points to, then increments that value.
+
+
+2. The left to right or right to left order for operator precedence is not guarenteed.
```

Comment:
right! The simple answer to this question is neither. C doesn’t always evaluate left-to-right or right-to-left. Generally, function calls are evaluated first, followed by complex expressions and then simple expressions.